curriculum/02-data-wrangling.md

```diff
@@ -0,0 +1,91 @@
+# Data Wrangling Curriculum
+
+## Table of Contents
+
+1. [Curriculum](#1-curriculum)
+2. [Topics Not Covered](#2-topics-not-covered)
+
+## 1 Curriculum
+
+| BLU   | Name | Specialization | Status   | Content           |
+|-------|------|----------------|----------|-------------------|
+| BLU01 | Messy data | Data Wrangling | Proposed | [Content](#blu01) |
+| BLU02 | Advanced wrangling  | Data Wrangling | Proposed | [Content](#blu02) |
+| BLU03 | Data sources | Data Wrangling | Proposed | [Content](#blu03) |
+
+## Contents
+
+### BLU01
+
+#### Messy data 
+
+##### Main topics
+
+1. Alternative tabular sources 
+1. Non tabular data sources 
```

Comment:
should read `Non-tabular`

Reply:
also numbering is strange, should be 1., 2., 3.

Reply:
this is actually the standard way to do numbered lists on markdown! but changed it anyways to be consistent with the others

Reply:
I only looked at the source, so if that would legit generate a numbered list, that would be fine then - I didn't know that it would be the standard :)

Reply:
I sounded really grumpy, I'm sorry 😂